src/catala/driver.ml

```diff
@@ -59,7 +58,7 @@ let driver (source_file : Pos.input_file) (debug : bool) (dcalc : bool) (unstyle
         Errors.raise_error
           (Printf.sprintf "The selected language (%s) is not supported by Catala" l)
     in
-    Cli.locale_lang := Cli.to_backend_lang language;
+    Cli.locale_lang := language;
```

Comment:
Just to be sure: if you omit the `--language` option, does it default to English ?

Reply:
There is no more default, because in my previous PR (#121) I added the language inference according the input file extension. For now, if the input file extension doesn't match any supported extensions (`.catala_en`, `.catala_fr` and `.catala_pl), an error is raised. Is it the desired behavior?